docs/rules/wrap-iife.md

```diff
@@ -18,12 +18,18 @@ This rule requires all immediately-invoked function expressions to be wrapped in
 
 ## Options
 
-The rule takes one option which can enforce a consistent wrapping style:
+This rule has two options, a string option and an object option.
```

Comment:
I think that this should be `This rule has two options: a string option and an object option.` or `This rule has a string and an object option.`